actionpack/lib/action_dispatch.rb

```diff
@@ -83,6 +83,7 @@ class MissingController < NameError
     autoload :RemoteIp
     autoload :ServerTiming
     autoload :ShowExceptions
+    autoload :TraceToFileExtractor
     autoload :SSL
     autoload :Static
```

Comment:
```suggestion
    autoload :SSL
    autoload :Static
    autoload :TraceToFileExtractor
```

Knit: to put the autoload at the end (in alphabetical order) also for a cleaner diff

Reply:
sorted